src/Connectors/src/Connector/PostgreSql/PostgreSqlWebApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Data;
+using System.Data.Common;
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Steeltoe.Common;
+using Steeltoe.Common.HealthChecks;
+
+namespace Steeltoe.Connector.PostgreSql;
+
+public static class PostgreSqlWebApplicationBuilderExtensions
+{
+    private const string BindingType = "postgresql";
+
+    public static WebApplicationBuilder AddPostgreSql(this WebApplicationBuilder builder)
```

Comment:
Why is this the only connector with a webApplicationBuilder extension? Do we in addition need a Iservicecollection.AddPostgreSql?

Reply:
Yes, definitely. Thanks for pointing this out. It's on my list of enhancements to address later. This PR is the bare minimum to make our PostgreSQL sample work, without changing the existing connectors' logic. That's also why I had to add a few ***2 types, to be renamed later.

Once I've added support for the other connectors, I'm going to refactor, clean up, document, etc.